src/index.html

```diff
@@ -11,9 +11,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>WHO Metadata App</title>
 
-    <link type="text/css" rel="stylesheet" href="../../../dhis-web-commons/material-design-icons/material-icons.css">   
-    <link type"text/css" rel="stylesheet" href='../../../dhis-web-commons/fonts/roboto.css'/>    
-    <link type"text/css" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link type="text/css" rel="stylesheet" href="../../../dhis-web-commons/material-design-icons/material-icons.css" />
+    <link type="text/css" rel="stylesheet" href='../../../dhis-web-commons/fonts/roboto.css'/>
+    <link type="text/css" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" />
```

Comment:
Possible to bundle these instead of hardcoding?

Reply:
Agree, I was just adding the self-closing tag but should have paid better attention to the href as well.